src/root.component.tsx

```diff
@@ -58,6 +59,15 @@ export function Root(props: NavProps) {
     }
   });
 
+  React.useEffect(() => {
+    const abortController = new AbortController();
+    openmrsFetch("/ws/rest/v1/appui/session", {
+      signal: abortController.signal
+    }).then(({ data }) => {
+      setCurrentSession(data);
+    });
+  });
```

Comment:
I think this will run on every render, no?
```suggestion
  }, []);
```